### ISSUE TITLE
Accessibility - External link opens in new tab is not conveyed to user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#2384]
## Detailed description

Issue :- 
There is an icon near these links which indicates that these links open in new window but, the same information is not conveyed  for the screen reader user.

Expected :
The information conveyed visually should be available for the screen reader user as well. Anyone of the following methods can be used to make it screen reader accessible.
1. provide the information in the link text. 
2. title attribute. or 
3. as an off screen text

Example: 
1. feed back form opens in new window
2. products opens external link

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/eba1f3a0-229e-4216-8d9b-bbc93b36cbfa)

**Comment by caiobozato** [comment]
I've talked with @RaphaelAmadeu and @kennylam and since this is something we don't have a native way to do, we need to discuss with all dev team and design as well.

My suggestion is to use something like `aria-label={"{link text} - This link opens in a new tab"}`

But again, we need to discuss that.

**Comment by asudoh** [comment]
Weighing in; I think this is a good idea. [More resource](https://www.w3.org/TR/WCAG20-TECHS/G200.html)

**Comment by RaphaelAmadeu** [comment]
I think that the only thing to consider is that as a Design System we'll have to support another languages. An idea would be an utility script for this purpose, maybe using the translation api

**Comment by ljcarot** [comment]
So we shouldn't assume that External links open in a new tab, their purpose is just to indicate leaving IBM to an "external page".  See v18 standard https://www.ibm.com/standards/web/v18/design/external-links/ for description.  Have we changed this behavior in Carbon?

**Comment by RaphaelAmadeu** [comment]
@ljcarot as far as I know, the previous QA guidelines pointed that all external links should be opened in another tab, we could confirm that

**Comment by ljcarot** [comment]
We should open up an additional knob in storybook to allow for open in new window or open in existing.  Then if we open in new window, screenreader will be able to detect that.  Dev story to work on.  cc: @jeffchew

**Comment by RaphaelAmadeu** [comment]
@ljcarot We just need to remember that opening external links in a new tab is set by code on `CTA` component itself, so this would require some editing on it, in order to be able to create a prop for target.
